invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessUsers/UserAccessSearchResult.js

```diff
@@ -39,26 +41,23 @@ class UserAccessSearchResultItem extends Component {
     const { deleteSuccessCallback } = this.props;
     deleteSuccessCallback(data);
   };
+
+  accessDropdownOptions = [
+    ...dropdownOptions,
+    { key: "manage", text: i18next.t("Can manage"), value: "manage" },
```

Comment:
Suggestion: remove the `can preview` option as I don't see any difference between `can_view` and `can_preview`